### ISSUE TITLE
Static filters don't seem to block remote fonts from 3rd party

### ISSUE THREAD

**Issue by antistress** [uBlockOrigin/uBlock-issues#323]
### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [ ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [ ] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

remotes fonts from 3rd party are not blocked despite dedicated filter (as suggested at https://github.com/gorhill/uBlock/wiki/Per-site-switches#no-remote-fonts).

Here are some of my filters : 
@@||ajax.googleapis.com^
||googleapis.com^
*$font,third-party

Note that, if I am not mistaken : 
- the 1st filter is there to allow Decentraleyes to do its job https://git.synz.io/Synzvato/decentraleyes/wikis/Frequently-Asked-Questions. And it's the only line I have in My Filters that start with "@@"
- the 2nd filter is there to block access to that Google domain (including fonts.googleapis.com but excluding ajax.googleapis.com - see 1st filter)
- the 3rd is there to block all remotes fonts from 3rd party

Also : 
- I'm using latest stable Firefox and uBlockOrigin
- uBlockOrigin advanced user features is not activated.

However both Lightbeam & uBlockOrigin add-ons show that connections are made to fonts.googleapis.com

![capture1](https://user-images.githubusercontent.com/2436957/49339964-c1375d80-f639-11e8-93af-88a47204073e.png)

![capture2](https://user-images.githubusercontent.com/2436957/49339963-c1375d80-f639-11e8-802c-e5a0be89f97d.png)

### A specific URL where the issue occurs

https://ledenicheur.fr/product.php?p=2798699

### Steps to Reproduce

1. uBlockOrigin advanced user features is not activated
2. add theses filters within uBlockOrigin My Filters pane : 
@@||ajax.googleapis.com^
||googleapis.com^
*$font,third-party
3. Go to https://ledenicheur.fr/product.php?p=2798699 and see that connection is made to fonts.googleapis.com

### Expected behavior:

No connection should be made to fonts.googleapis.com

### Actual behavior:

Connections are made to fonts.googleapis.com despite dedicated settings

### Your environment

* uBlock Origin version: 1.17.4
* Browser Name and version: Firefox 63.0.3 (64 bits
* Operating System and version: GNU/Linux Debian 64 bits Sid with GNOME

**Comment by antistress** [comment]
[mes-filtres-statiques-ublock_2018-12-02_14.03.49.txt](https://github.com/uBlockOrigin/uBlock-issues/files/2636932/mes-filtres-statiques-ublock_2018-12-02_14.03.49.txt)

I've attached all my filters

Thanks

**Comment by gwarser** [comment]
Dynamic URL filtering rule? Like this one:

`ledenicheur.fr https://fonts.googleapis.com stylesheet allow`

No need for advanced user mode. Check [logger](https://github.com/gorhill/uBlock/wiki/The-logger)

**Comment by gorhill** [comment]
Unable to reproduce, using default settings and lists + OP's filters.

@antistress you need to look at all your other settings and as always, use the logger to diagnose such issues.

**Comment by uBlock-user** [comment]
> Dynamic URL filtering rule? 

He hasn't activated dynamic filtering from the picture he posted.

**Comment by gorhill** [comment]
Also, @antistress, your filters are not very efficient, there is no need to prepend an asterisk to most of them , this prevent uBO from efficiently storing these filters.

**Comment by gorhill** [comment]
> He hasn't activated dynamic filtering from the picture he posted.

> No need for advanced user mode.

**Comment by antistress** [comment]
Hi again, 

To complete the discussion on this report, please note that 
*$font,third-party,important
**is not sufficient** to block fonts.googleapis.com that Adguard Base Filters allows.

Only 
||fonts.googleapis.com^$important
succeed in blocking fonts.googleapis.com despite Adguard Base Filters

**Comment by gorhill** [comment]
Yes, because `fonts.googleapis.com` is also used to serve CSS, which won't be blocked by `*$font,third-party,important`.

**Comment by uBlock-user** [comment]
`$font` only covers font type requests, fonts are also loaded over by `$stylesheets`, so it's better you block the entire domain.
